RELEASE.md

```diff
@@ -2,25 +2,35 @@
 TBD
 
 # Manual releases
-1. Set BUILD_VERSION, e.g. 1.0.0-beta.3. Ensure this tag exists for contiv-ui repo, else step 2 will fail.
+1. Check out the right branch and the right commit. This is necessary
+when not releasing from the HEAD of master.
+
+2. Tag the right commit and push it to GitHub. This is mandatory if the
+release isn't made from the HEAD of master.
+	```
+	git tag 1.0.1 3aba546aea1235
+	git push origin 1.0.1
+	```
+
+3. Set BUILD_VERSION, e.g. 1.0.0-beta.3. Ensure this tag exists for contiv-ui repo, else step 2 will fail.
 
 	```
 	export BUILD_VERSION=<build version>
 	```
 
-2. Build docker binary image. This would create a docker image contiv/auth_proxy:$BUILD_VERSION.
+4. Build docker binary image. This would create a docker image contiv/auth_proxy:$BUILD_VERSION.
```

Comment:
FYI, in Markdown you can just use `1.` over and over and it will automatically be numbered correctly in the output.  This also means you can reorganize sections without having to update their numbers